src/RealFourierArray.jl

```diff
@@ -18,10 +18,12 @@ A structure to facilitate manipulations of an array of real-space type T, in bot
 and fourier space. Create with `from_real` or `from_fourier`, and access
 with `A.real` and `A.fourier`.
 """
-mutable struct RealFourierArray{T <: Real}
+mutable struct RealFourierArray{T <: Real,
+                                Treal <: AbstractArray{T, 3},
```

Comment:
`Treal` ... not sure. Have not used that to mean `real(T)` at some place? How about `RealArray` and `ComplexArray`?